SpatialGDK/Documentation/content/get-started/build-unreal-fork.md

```diff
@@ -34,42 +47,65 @@ To get access to the SpatialOS Unreal Engine fork, you need to link your GitHub
 
-### Step 4: Build the Unreal Engine fork
+### Step 4: Prepare the fork
 
-To build the Unreal Engine fork: 
+To prepare the fork: 
 
-1. Run **Setup.bat**, found in the root directory of your clone of Unreal Engine.
-2. In the same directory, double-click **GenerateProjectFiles.bat**. This file automatically sets up the project files required to build Unreal Engine.<br/>
-3. Double-click **InstallGDK.bat**
-This automatically opens a command line window and performs the following:
-	* Clones the UnrealGDK into your Engine's `Plugins` directory
-	* Clones the [UnrealGDKExampleProject](https://github.com/spatialos/UnrealGDKExampleProject) into your Engine's `Samples` directory
-	* Runs the Unreal GDK `Setup.bat` script to install the GDK into the cloned `UnrealGDKExampleProject` directory
-	* Generates Visual Studio solution files for the `UnrealGDKExampleProject`<br/>
-This process can take a long time to complete. The command line window closes when the process has finished.    <br/>
-1. In the same directory, open **UE4.sln** in Visual Studio.
-2. In Visual Studio, on the toolbar, navigate to **Build** > **Configuration Manager**; set your active solution configuration to **Development Editor** and your active solution platform to **Win64**.
-3. In the Solution Explorer window, right-click on the **UE4** project and select **Set as StartUp Project**
-4. In the Solution Explorer window, right-click on the **UE4** project and select **Build** (you may be prompted to install some dependencies first). <br>
+1. Run Unreal's `setup.bat` script. </br>
+This is part of [Unreal's UE set-up](https://docs.unrealengine.com/en-US/GettingStarted/DownloadingUnrealEngine/index.html); it downloads binary content for UE, as well as installing prerequisites and setting up Unreal file associations. </br>
+To do this: </br>
+In File Explorer, navigate to the root directory of your clone of the SpatialOS Unreal Engine fork and double-click **Setup.bat**.
+2. Run Unreal's `GenerateProjectFiles.bat` script  - also part of Unreal's UE set-up. 
+</br> To do this: </br>
+In the same directory, double-click **GenerateProjectFiles.bat**.
 
-Visual Studio then builds Unreal Engine, which can take up to a couple of hours.
+### Step 5: Clone and install the plugin
+You need to clone the SpatialOS GDK plugin and install it in the UE fork and Example Project directory. You can follow either auto-install or manual-install, we recommend the auto-install.
 
-When the build is complete, you can continue to **3: Set up a project...**
+> **TIP:** Use auto-install as this makes setting up the Example Project and Starter Template quicker. You will be able to follow tutorials based on the Example Project more quickly. If you follow manual-install, you will need to take extra steps to set up the Example Project or Starer Template and follow tutorials.
```

Comment:
We used "Tip" above but "TIP" here

Reply:
That you spotted this makes me so proud!!!

Reply:
Our standard is lower case, so will amend.

Reply:
```suggestion
> **Tip:** Use auto-install as this makes setting up the Example Project and Starter Template quicker. You will be able to follow tutorials based on the Example Project more quickly. If you follow manual-install, you will need to take extra steps to set up the Example Project or Starer Template and follow tutorials.
```